components/KitchenSink.tsx

```diff
@@ -195,6 +196,74 @@ export const KitchenSink: React.FC = (): JSX.Element => {
             <Card style={styles.card}>
                 <Card.Title title="Grades" />
```

Comment:
Chips Card went under the Grades card title. This may have happened while resolving the merge conflicts. 
Please add separate Card for Chips.

![Simulator Screenshot - iphone 12 pro - 2023-11-30 at 17 42 18](https://github.com/etn-ccis/blui-react-native-showcase-demo/assets/120575281/f80b4ffe-7d47-4e69-a795-342bd93851f2)